### PR TITLE
Update bactopia to 1.2.4

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`bactopia`](https://bioconda.github.io/recipes/bactopia/README.html): **1.2.2** &rarr; **1.2.4**

[install with bioconda](http://bioconda.github.io/recipes/bactopia/README.html) [Conda](https://anaconda.org/bioconda/bactopia/files)

Info | Link
-----|-----
Recipe | [`recipes/bactopia`](https://github.com//bioconda/bioconda-recipes/tree/bump/bactopia/recipes/bactopia) (click to view/edit other files)
Releases |[https://github.com/bactopia/bactopia/tags](https://github.com/bactopia/bactopia/tags)
Author | `@bactopia`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by rpetit3** [comment]
Any recommendations on how to go about getting this fixed? The conda requirement was needed to use Nextflow's conda functions (e.g. creating environments). The last time I checked (months ago), Nextflow complained about conda missing, I can test again.

**Comment by dpryan79** [comment]
I vaguely remember something about docker testing being disabled if conda is a dependency. Do you remember anything about that @bgrurning ?

**Comment by rpetit3** [comment]
Looking into the conda dependency. The requirement is related to https://github.com/nextflow-io/nextflow/issues/1195#

I could potentially add this https://github.com/nextflow-io/nextflow/issues/1195#issuecomment-535396231 to the `build.sh`, but would like your opinion before doing so.

Otherwise, I think the conda dependency would need to remain until its sorted on Nextflow's end.

If the conda dependency is dropped, I can adjusts the tests to that the Docker image builds. The failed build was due to default nextflow outputs.

Tagging @bgruening (small typo in comment above)

**Comment by rpetit3** [comment]
Ok, I've applied the suggested fix for https://github.com/nextflow-io/nextflow/issues/1195#issuecomment-535396231 on an install to my personal conda channel

I've added the following to the `build.sh`
```
# Help Nextflow find activate (for conda activate)
ln -s `which activate` ${PREFIX}/bin/activate
```

This seems to allow Nextflow to use conda environments and would not longer require conda as a dependency in the `meta.yml`

Is this fix ok? If so, I'll update the recipe.

**Comment by johanneskoester** [comment]
> Ok, I've applied the suggested fix for [nextflow-io/nextflow#1195 (comment)](https://github.com/nextflow-io/nextflow/issues/1195#issuecomment-535396231) on an install to my personal conda channel
> 
> I've added the following to the `build.sh`
> 
> ```
> # Help Nextflow find activate (for conda activate)
> ln -s `which activate` ${PREFIX}/bin/activate
> ```
> 
> This seems to allow Nextflow to use conda environments and would not longer require conda as a dependency in the `meta.yml`
> 
> Is this fix ok? If so, I'll update the recipe.

I don't think this is a good idea. It is hard to predict what kind of side effects such a hack would have. I think it would be better to fix conda support in nextflow. 

cc @pditommaso

**Comment by rpetit3** [comment]
I agree, not a fan of it either.

For the time being, would it be ok to maintain the conda dependency in bactopia and manually merge future PRs after a successful build?

**Comment by johanneskoester** [comment]
I guess this depends a bit on how hard it would be to fix nextflow and how long it would take.

**Comment by mbargull** [comment]
> I vaguely remember something about docker testing being disabled if conda is a dependency.

We shouldn't have to have that restriction anymore since the container image we use has `conda>=4.4`.
Removing that restriction in : https://github.com/bioconda/bioconda-utils/pull/629

File: recipes/bactopia/meta.yaml [hunk 1]
```diff
@@ -10,9 +10,10 @@ build:
 
 source:
   url: https://github.com/bactopia/bactopia/archive/v{{version}}.tar.gz
-  sha256: ccdc1cdc6ce9123c38baec634b0dfa63462a155962141919d7d98aed118da334
+  sha256: 00d9422d2343b3e66d777651aad2c1b7632cd33aa63970ed8a0e350af5469ab7
 
 requirements:
+  host:
```
Comment (johanneskoester) [inline recipes/bactopia/meta.yaml]: The empty host section can be removed.

**Pull request merged**